blockstack-sdk/src/main/java/org/blockstack/android/sdk/ui/ConnectActivity.kt

```diff
@@ -0,0 +1,95 @@
+package org.blockstack.android.sdk.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.annotation.StyleRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.coroutineScope
+import kotlinx.android.synthetic.main.activity_connect.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.launch
+import org.blockstack.android.sdk.BlockstackConnect
+import org.blockstack.android.sdk.R
+
+class ConnectActivity : AppCompatActivity() {
+
+    private val LAUNCH_HOW_IT_WORKS_ACTIVITY = 1
```

Comment:
This should probably go to a `companion object`. And it could be named more closely with its purpose: `REQUEST_HOW_IT_WORKS`.